graalvm-nativeimage/quarkus/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+
+quarkus.datasource.db-kind=oracle
+quarkus.datasource.jdbc.driver=oracle.jdbc.driver.OracleDriver
+quarkus.datasource.jdbc.url=jdbc:oracle:thin:@gdpaulsep27i_tp?TNS_ADMIN=/Users/pparkins/Downloads/Wallet_gdpaulsep27o
```

Comment:
Can you pass that env variable like a HTTP parameter? I never knew that!

Reply:
yep :)